app/src/test/kotlin/net/rafaeltoledo/social/MainActivityTest.kt

```diff
@@ -1,16 +1,29 @@
 package net.rafaeltoledo.social
 
-import org.junit.Assert.assertNotNull
+import android.widget.TextView
+import com.google.common.truth.Truth.assertThat
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.robolectric.Robolectric
 import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
 
 @RunWith(RobolectricTestRunner::class)
 class MainActivityTest {
 
     @Test
     fun checkIfActivityIsSuccessfullyCreated() {
-        assertNotNull(Robolectric.setupActivity(MainActivity::class.java))
+        // Arrange
+        val newValue = "Test Social App"
+        val app = RuntimeEnvironment.application as TestSocialApp
+        app.initKoin(newValue)
+
+        // Act
+        val activity = Robolectric.setupActivity(MainActivity::class.java)
+
+        // Assert
+        assertThat(activity).isNotNull()
```

Comment:
I don't think there is much value in this assertion. If it is null it will fail on the line below with a very clear NPE so I'd just skip this assertion.